templates/form-fields/file-field.php

```diff
@@ -32,19 +32,20 @@
 <div class="wpum-uploaded-files">
 	<?php if ( ! empty( $data->value ) ) : ?>
 		<?php if ( is_array( $data->value ) ) : ?>
-			<?php foreach ( $data->value as $value ) : ?>
+			<?php 	if (isset($data->value['url']) || array_key_exists('url', $data->value)) {
```

Comment:
Why do an isset or array_key_exists?

Reply:
@singhrvijay23 the isset is enough. The call to array_key_exists will only be reached if the url key is present but has a null value, in which case there is no need to check again if the key exists